script

```diff
@@ -2727,155 +2602,54 @@ only_send_error_notifications="0"
 
         # find each vdisk extension and use it to build a regular expression to get a vdisk's trailing number.
         for extension in "${vdisk_extensions[@]}"
-
         do
+          if [ "$inline_zstd_compress" -eq 1 ]; then
```

Comment:
If we add this extension during the creation of the array as noted earlier, we can remove this if statement.

Reply:
This should be fixed, although I didn't end up adding it to the creation of the extensions array as I believe it is still used a few other places that need to continue to refer to the source extension(s).